rasterio/merge.py

```diff
@@ -125,11 +125,6 @@ def merge(
             min: pixel-wise min of existing and new
             max: pixel-wise max of existing and new
         or custom callable with signature:
-
-        def function(merged_data, new_data, merged_mask, new_mask, index=None, roff=None, coff=None):
-
-            Parameters
-            ----------
```

Comment:
Can you explain this change?

Reply:
It's not really being formatted correctly: https://rasterio.readthedocs.io/en/latest/api/rasterio.merge.html#rasterio.merge.merge

I don't think this kind of text is possible inside of a parameter section. A better way to document it might be to put it in the docstring above the parameters section and then point to it inside of parameters. Nested parameters inside of parameters just isn't supported.